relnotes

```diff
@@ -429,7 +428,7 @@ EOF+
     echo "### Previous Releases Included in $release_tag"
     while read anchor; do
       echo "$anchor"
-    done< <(egrep "^- \[${release_tag}-" $changelog)
```

Comment:
this doesn't seem to have equivalent output:
```
$ egrep "^- \[v1.5.0-" $(git rev-parse --show-toplevel)/CHANGELOG.md
- [v1.5.0-beta.3](#v150-beta3)
- [v1.5.0-beta.2](#v150-beta2)
- [v1.5.0-beta.1](#v150-beta1)
- [v1.5.0-alpha.2](#v150-alpha2)
- [v1.5.0-alpha.1](#v150-alpha1)
- [v1.5.0-beta.1](CHANGELOG.md#v150-beta1)
- [v1.5.0-alpha.2](CHANGELOG.md#v150-alpha2)
- [v1.5.0-alpha.1](CHANGELOG.md#v150-alpha1)
$ git tag --sort=version:refname | egrep "^v1.5.0-"
v1.5.0-alpha.0
v1.5.0-alpha.1
v1.5.0-alpha.2
v1.5.0-beta.0
v1.5.0-beta.1
v1.5.0-beta.2
v1.5.0-beta.3
```

Notably:
* the order is different
* using `CHANGELOG.md` produces a list and anchor links, while using the tag doesn't do either.

Were these changes intentional?